docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -234,6 +239,14 @@ log-file: "teku_2020-01-01.log"
 
 Relative or absolute location, and filename of the log file.
 
+The default directory is OS dependant:
+
+* macOS: `~/Library/teku/logs`
+* Unix/Linux: `~/.teku/logs`
```

Comment:
sorry, if the data directory above is set to /root/.local/share/teku, I'm guessing the logs dir here should be similar - default config would have it at `/root/.local/share/teku/logs`